### DISCUSSION TITLE
Rename site folder name

### DISCUSSION THREAD

**Discussion by fifisave** [leokhoa/laragon#820] [General]
Hello I installed a full WordPress website and theme locally using Laragon 6.0 BUT the initial name of my folder site MUST be "horue" and not "horus"...... So I must rename it instead of reinstalling everything !!
How can I do that without losing everything, please ?
Regards

**Comment by slolo2000** [comment]
Hello,

Normally you can do it with a Wordpress plugin like "Better Search Replace" :
See: https://wordpress.org/plugins/better-search-replace/

This will change your website URL in all Wordpress sources.
After that, you should be able to rename your folder.

If you have error messages after the operation, you may need to modify the Wordpress configuration files or modify the database values ​​with phpMyAdmin.
